spec/features/defence_request_creation_spec.rb

```diff
@@ -91,6 +91,36 @@
    expect(page).to have_content 'No results found'
   end
 
+  scenario "using Close link to clear solicitor search results", js: true do
+    stub_solicitor_search_for_bob_smith
+     visit root_path
+     click_link 'New Defence Request'
+     choose 'Own'
+     fill_in 'q', with: "Bob Smith"
+
+     click_button 'Search'
+     expect(page).to have_content 'Bobson Smith'
+
+     within('.solicitor_results_list') do
+      click_link("Close")
```

Comment:
yeah not sure whose editor is not set to 2 spaces, was something I was going to raise